src/main/java/com/spotinst/sdkjava/example/statefulnode/azure/AzureStatefulNodeExample.java

```diff
@@ -0,0 +1,849 @@
+        StatefulNodeRevertToSpotConfiguration revertToSpot =
+                StatefulNodeRevertToSpotConfiguration.Builder.get().setPerformAt(AzureStatefulNodePerformAtEnum.ALWAYS).build();
+        //Build Strategy
+        StatefulNodeStrategyConfiguration.Builder strategyBuilder = StatefulNodeStrategyConfiguration.Builder.get();
+        StatefulNodeStrategyConfiguration strategy =
+                strategyBuilder.setSignals(signalList).setFallbackToOd(true).setDrainingTimeout(180).setPreferredLifecycle(
+                        AzureStatefulNodeLifeCycleTypeEnum.OD)
+                               .setRevertToSpot(revertToSpot).setOptimizationWindows(optimizationWindows).build();
+
+        //Build Scheduling Tasks
+        StatefulNodeTasksConfiguration.Builder tasksBuilder1 = StatefulNodeTasksConfiguration.Builder.get();
+        StatefulNodeTasksConfiguration task1 =
+                tasksBuilder1.setIsEnabled(true).setCronExpression("0 1 * * *").setType(AzureStatefulNodeTaskTypeEnum.PAUSE).build();
+
+        StatefulNodeTasksConfiguration.Builder tasksBuilder2 = StatefulNodeTasksConfiguration.Builder.get();
+        StatefulNodeTasksConfiguration task2 =
+                tasksBuilder2.setIsEnabled(false).setCronExpression("37 * * * *").setType(AzureStatefulNodeTaskTypeEnum.RESUME).build();
+        List<StatefulNodeTasksConfiguration> tasksList = new ArrayList<>();
+        tasksList.add(task1);
+        tasksList.add(task2);
+
+        //Build Scheduling
+        StatefulNodeSchedulingConfiguration scheduling =
+                StatefulNodeSchedulingConfiguration.Builder.get().setTasks(tasksList).build();
+
+        // Build persistent
+        StatefulNodePersistenceConfiguration.Builder persistentBuilder = StatefulNodePersistenceConfiguration.Builder.get();
+        StatefulNodePersistenceConfiguration persistent =
+                persistentBuilder.setShouldPersistDataDisks(false).setShouldPersistNetwork(true).setShouldPersistOsDisk(false).setDataDisksPersistenceMode(
+                        AzureStatefulNodeDiskModeEnum.ONLAUNCH)
+                                 .setOsDiskPersistenceMode(AzureStatefulNodeDiskModeEnum.ONLAUNCH).build();
+
+        //Build Health
+        List<HealthCheckTypeEnumAzure> healthCheckTypesList = new ArrayList<>();
+        healthCheckTypesList.add(HealthCheckTypeEnumAzure.VM_STATE);
+        StatefulNodeHealthConfiguration.Builder healthBuilder = StatefulNodeHealthConfiguration.Builder.get();
+        StatefulNodeHealthConfiguration health =
+                healthBuilder.setHealthCheckTypes(healthCheckTypesList).setUnhealthyDuration(300).setGracePeriod(180).setAutoHealing(true).build();
+
+        // Build Stateful Node
+        StatefulNode.Builder statefulNodeBuilder = StatefulNode.Builder.get();
+        StatefulNode statefulNode =
+                statefulNodeBuilder.setName("Automation-java-SDK-StatefulNode").setRegion("eastus").setResourceGroupName("ManualQAResourceGroup")
```

Comment:
Automation should be using resources from AutomationResourceGroup. Please change throughout the examples.